tools/htdocs/components/20_VEPForm.js

```diff
@@ -155,6 +155,8 @@ Ensembl.Panel.VEPForm = Ensembl.Panel.ToolsForm.extend({
     if (!val) {
       return;
     }
+    // Remove potential new line and carriage return characters
+    val = val.replace(/[\r\n]g/, '');
```

Comment:
Typo in the regex. Please could you fix this?

Reply:
Well spotted! Laurent said it worked on his sandbox, so I assumed the code was OK, but that 'g' needs to go after the slash, right?

Reply:
How on earth did I write this typo in my PR ?
I will send a fix. Sorry about that!